src/controller/java/AndroidDeviceControllerWrapper.cpp

```diff
@@ -49,6 +53,15 @@ void AndroidDeviceControllerWrapper::CallJavaMethod(const char * methodName, jin
     CallVoidInt(JniReferences::GetEnvForCurrentThread(), mJavaObjectRef, methodName, argument);
 }
 
+CHIP_ERROR AndroidDeviceControllerWrapper::GetRootCACertificate(chip::FabricId fabricId, uint8_t * certBuf, uint32_t certBufSize,
+                                                                uint32_t & outCertLen)
+{
+    Initialize();
```

Comment:
I wonder if this should go to java code instead of creating a new root cert inline.

Reply:
It definitely should, but that can be done as a separate change I think